.github/workflows/pr.yml

```diff
@@ -19,42 +53,57 @@ jobs:
           - ubuntu-latest
           - [runs-on, runner=8cpu-linux-arm64, "run-id=${{ github.run_id }}"]
           - macos-latest
-          - macos-13
     runs-on: ${{ matrix.runner }}
     timeout-minutes: 15
     env:
+      ATHENA_DIR: "${{ github.workspace }}/.athena"
       CARGO_NET_GIT_FETCH_WITH_CLI: "true"
     steps:
       - name: Checkout sources
         uses: actions/checkout@v4
 
-      - name: Setup CI
-        uses: ./.github/actions/setup
+      - name: Setup Rust toolchain
+        uses: actions-rust-lang/setup-rust-toolchain@v1
+
+      - name: Install Athena toolchain
+        run: |
+          curl -L https://install.athenavm.org | bash
+          $ATHENA_DIR/bin/athup --path $GITHUB_WORKSPACE
+          $ATHENA_DIR/bin/cargo-athena athena --version
+          echo "$ATHENA_DIR/bin" >> $GITHUB_PATH
+
+      - name: Install Athena CLI
```

Comment:
As mentioned above: you don't need to _both_ install and run athup, and also run `cargo install cli`. Remove one or the other. If the goal here is to run the _local_ version of the CLI, let's drop "Install Athena toolchain" in this job entirely.

Reply:
fixed